theano/sandbox/gpuarray/elemwise.py

```diff
@@ -576,8 +604,8 @@ class GpuCAReduceCuda(HideC, CAReduceDtype):
 
     pre_scalar_op: if present, must be a scalar op with only 1
     input. We will execute it on the input value before reduction.
-
     """
+    _f16_ok = True
```

Comment:
Well apparently that code is in another branch.  But since I'm tired to play find-the-commit and I keep mixing what is where, I just pulled the extra commits from my working branch to this one.  So there should be some code changes now.